11-08 00:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 01:01:00 [INFO] zhenxun | CMD[定时任务] 自动清理临时数据完成，共清理了 20.19MB 的数据...
11-08 01:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 02:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 02:06:26 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.private.friend]: Message 1804635829 from 2747277822 '[json:data={"app":"com.tencent.gamecenter.mall","desc":"【宝可梦大集结-今日上线】线下4城见面会活动...]'
11-08 02:06:26 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 02:06:26 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 02:06:26 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 02:06:26 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 02:06:26 [ERROR] nonebot | Running Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) failed.
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x0000026058C61510>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x0000026058C61360>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000002605943ED40>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 577, in run
    server.run()
    │      └ <function Server.run at 0x000002605943E680>
    └ <uvicorn.server.Server object at 0x00000260689C63B0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000002605943E710>
           │       │   └ <uvicorn.server.Server object at 0x00000260689C63B0>
           │       └ <function run at 0x000002605695CE50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x0000026068F758C0>
           │    └ <function BaseEventLoop.run_until_complete at 0x0000026056A60280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x00000260586A1510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x0000026056A61CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x0000026056A109D0>
    └ <Handle Task.task_wakeup(<_GatheringFu...b400876'}})]]>)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle Task.task_wakeup(<_GatheringFu...b400876'}})]]>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle Task.task_wakeup(<_GatheringFu...b400876'}})]]>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle Task.task_wakeup(<_GatheringFu...b400876'}})]]>)>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
          └ <function _run_matcher at 0x0000026058DC09D0>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _event_to_me at 0x0000026058CB9CF0>: <Task finished name='Task-15678123' coro=<_event_to_me() done, defined at C:\...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x000002606CAD4F10>
          │       │   │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │       │   │    └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │       │   └ Bot(type='OneBot V11', self_id='3973508735')
          │       └ <function Matcher.run at 0x0000026058CB9B40>
          └ Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _event_to_me at 0x0000026058CB9CF0>: <Task finished name='Task-15678123' coro=<_event_to_me() done, defined at C:\...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x000002606CAD4F10>
          │    │          │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │    │          │    └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │    │          └ Bot(type='OneBot V11', self_id='3973508735')
          │    └ <function Matcher.simple_run at 0x0000026058CB9AB0>
          └ Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 825, in simple_run
    await handler(
          └ Dependent(call=_)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
                 │    │             │         │    │    │       └ {'session': Session(bot_id='3973508735', bot_type='OneBot V11', platform='qq', level=<SessionLevel.LEVEL1: 1>, id1='274727782...
                 │    │             │         │    │    └ <function _ at 0x0000026069352170>
                 │    │             │         │    └ Dependent(call=_)
                 │    │             │         └ ~R
                 │    │             └ <class 'collections.abc.Awaitable'>
                 │    └ typing.Callable
                 └ <function cast at 0x00000260567F04C0>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\parse_bilibili\__init__.py", line 86, in _
    get_url = (qqdocurl_value if qqdocurl_value else jumpUrl_value).split("?")[
               │                 │                   └ {}
               │                 └ {}
               └ {}

AttributeError: 'dict' object has no attribute 'split'
11-08 02:06:27 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 02:06:27 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 02:06:27 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 02:06:27 [INFO] zhenxun | begin to build uni message!!!
11-08 02:06:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 02:06:37 [ERROR] nonebot | Running Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) failed.
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x0000026058C61510>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x0000026058C61360>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000002605943ED40>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 577, in run
    server.run()
    │      └ <function Server.run at 0x000002605943E680>
    └ <uvicorn.server.Server object at 0x00000260689C63B0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000002605943E710>
           │       │   └ <uvicorn.server.Server object at 0x00000260689C63B0>
           │       └ <function run at 0x000002605695CE50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x0000026068F758C0>
           │    └ <function BaseEventLoop.run_until_complete at 0x0000026056A60280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x00000260586A1510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x0000026056A61CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x0000026056A109D0>
    └ <Handle Task.task_wakeup(<Future finished result=None>)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle Task.task_wakeup(<Future finished result=None>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle Task.task_wakeup(<Future finished result=None>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle Task.task_wakeup(<Future finished result=None>)>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
          └ <function _run_matcher at 0x0000026058DC09D0>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _event_to_me at 0x0000026058CB9CF0>: <Task finished name='Task-15678123' coro=<_event_to_me() done, defined at C:\...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x000002606CAD4F10>
          │       │   │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │       │   │    └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │       │   └ Bot(type='OneBot V11', self_id='3973508735')
          │       └ <function Matcher.run at 0x0000026058CB9B40>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _event_to_me at 0x0000026058CB9CF0>: <Task finished name='Task-15678123' coro=<_event_to_me() done, defined at C:\...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x000002606CAD4F10>
          │    │          │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │    │          │    └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
          │    │          └ Bot(type='OneBot V11', self_id='3973508735')
          │    └ <function Matcher.simple_run at 0x0000026058CB9AB0>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 825, in simple_run
    await handler(
          └ Dependent(call=_)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
                 │    │             │         │    │    │       └ {'session_id': 'qq_OneBot_V11_3973508735_2747277822', 'arg': [MessageSegment(type='json', data={'data': '{"app":"com.tencent....
                 │    │             │         │    │    └ <function _ at 0x0000026067D23B50>
                 │    │             │         │    └ Dependent(call=_)
                 │    │             │         └ ~R
                 │    │             └ <class 'collections.abc.Awaitable'>
                 │    └ typing.Callable
                 └ <function cast at 0x00000260567F04C0>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\nonebot_plugin_sparkapi\matchers\chat.py", line 58, in _
    await hello().finish()
          └ <function hello at 0x0000026067D23E20>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_alconna\uniseg\message.py", line 1407, in finish
    await self.send(target, bot, fallback, at_sender, reply_to, **kwargs)
          │    │    │       │    │         │          │           └ {}
          │    │    │       │    │         │          └ False
          │    │    │       │    │         └ False
          │    │    │       │    └ <FallbackStrategy.rollback: 'rollback'>
          │    │    │       └ None
          │    │    └ None
          │    └ <function UniMessage.send at 0x000002605BB49480>
          └ [Image(id=None, url=None, path=WindowsPath('resources/image/zai/5.jpg'), raw=None, mimetype=None, name='5.jpg'), Text(text='我...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_alconna\uniseg\message.py", line 1395, in send
    res = await fn.send_to(target, bot, msg, **kwargs)
                │  │       │       │    │      └ {}
                │  │       │       │    └ [MessageSegment(type='image', data={'file': 'file:///C:/Program%20Files/zhenxun/zhenxun_bot-dev/resources/image/zai/5.jpg', '...
                │  │       │       └ Bot(type='OneBot V11', self_id='3973508735')
                │  │       └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
                │  └ <function Onebot11MessageExporter.send_to at 0x000002605BB2C820>
                └ <nonebot_plugin_alconna.uniseg.adapters.onebot11.exporter.Onebot11MessageExporter object at 0x000002605BAEA8C0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_alconna\uniseg\adapters\onebot11\exporter.py", line 154, in send_to
    return await bot.send(target, message, **kwargs)  # type: ignore
                 │   │    │       │          └ {}
                 │   │    │       └ [MessageSegment(type='image', data={'file': 'file:///C:/Program%20Files/zhenxun/zhenxun_bot-dev/resources/image/zai/5.jpg', '...
                 │   │    └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
                 │   └ <function Bot.send at 0x00000260591E8160>
                 └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\bot.py", line 237, in send
    return await self.__class__.send_handler(self, event, message, **kwargs)
                 │    │                      │     │      │          └ {}
                 │    │                      │     │      └ [MessageSegment(type='image', data={'file': 'file:///C:/Program%20Files/zhenxun/zhenxun_bot-dev/resources/image/zai/5.jpg', '...
                 │    │                      │     └ PrivateMessageEvent(time=1731002786, self_id=3973508735, post_type='message', sub_type='friend', user_id=2747277822, message_...
                 │    │                      └ Bot(type='OneBot V11', self_id='3973508735')
                 │    └ <attribute '__class__' of 'object' objects>
                 └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\bot.py", line 190, in send
    return await bot.send_msg(**params)
                 │              └ {'user_id': 2747277822, 'message_type': 'private', 'message': [MessageSegment(type='image', data={'file': 'file:///C:/Program...
                 └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\adapter\bot.py", line 122, in call_api
    raise exception
          └ ActionFailed(status='failed', retcode=1200, data=None, message='Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgServic...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\adapter\bot.py", line 97, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'user_id': 2747277822, 'message_type': 'private', 'message': [MessageSegment(type='image', data={'file': 'file:///C:/Program...
                   │    │       │         │     └ 'send_msg'
                   │    │       │         └ Bot(type='OneBot V11', self_id='3973508735')
                   │    │       └ <function Adapter._call_api at 0x0000026056518820>
                   │    └ Adapter(name='OneBot V11')
                   └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\adapter.py", line 147, in _call_api
    return handle_api_result(await self._result_store.fetch(seq, timeout))
           │                       │    │             │     │    └ 30.0
           │                       │    │             │     └ 3707
           │                       │    │             └ <function ResultStore.fetch at 0x00000260591E92D0>
           │                       │    └ <nonebot.adapters.onebot.store.ResultStore object at 0x0000026058F0C5E0>
           │                       └ Adapter(name='OneBot V11')
           └ <function handle_api_result at 0x0000026059170280>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\utils.py", line 58, in handle_api_result
    raise ActionFailed(**result)
          │              └ {'status': 'failed', 'retcode': 1200, 'data': None, 'message': 'Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgServic...
          └ <class 'nonebot.adapters.onebot.v11.exception.ActionFailed'>

nonebot.adapters.onebot.v11.exception.ActionFailed: ActionFailed(status='failed', retcode=1200, data=None, message='Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{}\n\n    at Timeout.sendDataCallback [as _onTimeout] (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7666:15)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)', wording='Error: Timeout: NTEvent serviceAndMethod:NodeIKernelMsgService/sendMsg ListenerName:NodeIKernelMsgListener/onMsgInfoListUpdate EventRet:\n{}\n\n    at Timeout.sendDataCallback [as _onTimeout] (file:///C:/Users/Administrator/Documents/LiteloaderQQNT/plugins/NapCat.Framework/napcat.mjs:7666:15)\n    at listOnTimeout (node:internal/timers:573:17)\n    at process.processTimers (node:internal/timers:514:7)', echo='3707')
11-08 03:01:00 [ERROR] apscheduler | Job "_ (trigger: cron[hour='3', minute='1'], next run at: 2024-11-09 03:01:00 CST)" raised an exception
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\nonebot_plugin_sparkapi\config.py", line 131, in _
    logger.info("清理AI日常聊天数据完成...", "spark")

NameError: name 'logger' is not defined


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x0000026058C61510>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x0000026058C61360>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000002605943ED40>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 577, in run
    server.run()
    │      └ <function Server.run at 0x000002605943E680>
    └ <uvicorn.server.Server object at 0x00000260689C63B0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000002605943E710>
           │       │   └ <uvicorn.server.Server object at 0x00000260689C63B0>
           │       └ <function run at 0x000002605695CE50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x0000026068F758C0>
           │    └ <function BaseEventLoop.run_until_complete at 0x0000026056A60280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x00000260586A1510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x0000026056A61CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x0000026056A109D0>
    └ <Handle <TaskStepMethWrapper object at 0x000002606CD5A710>()>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x000002606CD5A710>()>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle <TaskStepMethWrapper object at 0x000002606CD5A710>()>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle <TaskStepMethWrapper object at 0x000002606CD5A710>()>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\apscheduler\executors\base_py3.py", line 30, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
                   │   │     │   │       │   └ <member 'kwargs' of 'Job' objects>
                   │   │     │   │       └ <Job (id=34ebb99a1fee4a6d8ea931ca46f89965 name=_)>
                   │   │     │   └ <member 'args' of 'Job' objects>
                   │   │     └ <Job (id=34ebb99a1fee4a6d8ea931ca46f89965 name=_)>
                   │   └ <member 'func' of 'Job' objects>
                   └ <Job (id=34ebb99a1fee4a6d8ea931ca46f89965 name=_)>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\plugins\nonebot_plugin_sparkapi\config.py", line 133, in _
    logger.error("清理AI日常聊天数据失败...", e=e)

NameError: name 'logger' is not defined
11-08 03:01:00 [INFO] zhenxun | 自动更新群组成员信息成功...
11-08 03:01:00 [INFO] zhenxun | 自动更新好友信息成功...
11-08 03:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 03:25:02 [INFO] zhenxun | CMD[自动备份] 自动备份成功...
11-08 04:01:00 [WARNING] zhenxun | 当前无UP池或 prts_up_char.json 文件不存在
11-08 04:01:00 [WARNING] zhenxun | 坎公骑冠剑UP更新出错 <class 'IndexError'>：list index out of range
11-08 04:01:00 [WARNING] zhenxun | guardian_up_char 解析出错
11-08 04:01:00 [WARNING] zhenxun | 赛马娘UP更新出错 || 错误<class 'AttributeError'>: 'NoneType' object has no attribute 'find'
11-08 04:01:00 [WARNING] zhenxun | pretty_up_char 解析出错
11-08 04:01:03 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者水
11-08 04:01:06 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者草
11-08 04:01:09 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者雷
11-08 04:01:12 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者岩
11-08 04:01:13 [WARNING] zhenxun | 原神 获取额外信息错误 旅行者风
11-08 04:01:15 [INFO] zhenxun | 原神 更新成功
11-08 04:01:15 [INFO] zhenxun | 原神 武器更新成功
11-08 04:01:16 [INFO] zhenxun | 成功获取原神当前up信息...当前up池: 「赤团开时」 & 「月草的赐慧」 & 「神铸赋形」
11-08 04:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 04:30:02 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/11/01/19/12/53/113049474_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\6.jpg
11-08 04:30:02 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/6.jpg --> /resources\image\_setu/6.jpg
11-08 04:30:02 [ERROR] zhenxun | 更新色图 6.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\6.jpg' -> 'resources\\image\\_setu\\6.jpg'
11-08 04:30:06 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2022/12/01/00/21/42/103235297_p0.jpg 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\5.jpg
11-08 04:30:06 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/5.jpg --> /resources\image\_setu/5.jpg
11-08 04:30:06 [ERROR] zhenxun | 更新色图 5.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\5.jpg' -> 'resources\\image\\_setu\\5.jpg'
11-08 04:30:34 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/03/29/00/00/36/106651983_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\4.jpg
11-08 04:30:36 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2022/04/04/18/18/36/97403745_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\3.jpg
11-08 04:30:36 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/3.jpg --> /resources\image\_setu/3.jpg
11-08 04:30:36 [ERROR] zhenxun | 更新色图 3.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\3.jpg' -> 'resources\\image\\_setu\\3.jpg'
11-08 04:30:41 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/10/27/18/20/27/112890527_p0.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\2.jpg
11-08 04:30:45 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/11/15/21/06/20/113432062_p0.jpg 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\1.jpg
11-08 04:30:45 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/1.jpg --> /resources\image\_setu/1.jpg
11-08 04:30:45 [ERROR] zhenxun | 更新色图 1.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\1.jpg' -> 'resources\\image\\_setu\\1.jpg'
11-08 04:30:47 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/04/18/19/12/19/107296589_p0.jpg 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\0.jpg
11-08 04:30:47 [INFO] zhenxun | 不需要压缩，移动图片resources\temp/0.jpg --> /resources\image\_setu/0.jpg
11-08 04:30:47 [ERROR] zhenxun | 更新色图 0.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\temp\\0.jpg' -> 'resources\\image\\_setu\\0.jpg'
11-08 04:30:52 [INFO] zhenxun | 下载 https://i.pixiv.re/img-original/img/2023/01/06/08/04/26/104277373_p3.png 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\0.jpg
11-08 04:30:52 [INFO] zhenxun | CMD[自动更新色图] ("2024-11-08 04:30:52 更新 色图 完成，本地存在 8 张，实际更新 3 张，以下为更新时未知错误：\n更新色图 6.jpg 错误 <class 'FileExistsError'>: [WinError 183] 当文件已存在时，无法创建该文件。: 'resources\\\\temp\\\\6.jpg' -> 'resources\\\\image\\\\_setu\\\\6.jpg'",)
11-08 04:40:00 [INFO] zhenxun | CMD[Chat检测] [Target]([992333365]) 群组两日内未发送任何消息，关闭该群全部被动
11-08 05:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 06:01:00 [INFO] zhenxun | begin to build uni message!!!
11-08 06:01:00 [INFO] zhenxun | group list is [<GroupConsole>, <GroupConsole>]
11-08 06:01:00 [INFO] zhenxun | key is 992333365:None, check_func is <function check at 0x0000026062109AB0>
11-08 06:01:00 [INFO] zhenxun | is_run value False
11-08 06:01:00 [INFO] zhenxun | key is 546881717:None, check_func is <function check at 0x0000026062109AB0>
11-08 06:01:00 [INFO] zhenxun | is_run value True
11-08 06:01:00 [INFO] zhenxun | target is Target({'id': '546881717', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
11-08 06:01:00 [INFO] zhenxun | begin to build uni message!!!
11-08 06:01:00 [INFO] zhenxun | 每日早安发送...
11-08 06:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 07:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 08:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 09:01:00 [INFO] zhenxun | begin to build uni message!!!
11-08 09:01:00 [INFO] zhenxun | group list is [<GroupConsole>, <GroupConsole>]
11-08 09:01:00 [INFO] zhenxun | key is 992333365:None, check_func is None
11-08 09:01:00 [INFO] zhenxun | is_run value True
11-08 09:01:00 [INFO] zhenxun | target is Target({'id': '992333365', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
11-08 09:01:00 [INFO] zhenxun | begin to build uni message!!!
11-08 09:01:00 [INFO] zhenxun | key is 546881717:None, check_func is None
11-08 09:01:00 [INFO] zhenxun | is_run value True
11-08 09:01:00 [INFO] zhenxun | target is Target({'id': '546881717', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
11-08 09:01:00 [INFO] zhenxun | begin to build uni message!!!
11-08 09:01:00 [INFO] zhenxun | 每日摸鱼发送...
11-08 09:01:03 [INFO] zhenxun | 下载 https://api.03c3.cn/api/zb 成功.. Path：C:\Program Files\zhenxun\zhenxun_bot-dev\resources\temp\moyu1731027660.jpeg
11-08 09:01:03 [INFO] zhenxun | begin to build uni message!!!
11-08 09:01:03 [INFO] zhenxun | group list is [<GroupConsole>, <GroupConsole>]
11-08 09:01:03 [INFO] zhenxun | key is 992333365:None, check_func is None
11-08 09:01:03 [INFO] zhenxun | is_run value True
11-08 09:01:03 [INFO] zhenxun | target is Target({'id': '992333365', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
11-08 09:01:03 [INFO] zhenxun | begin to build uni message!!!
11-08 09:01:03 [INFO] zhenxun | key is 546881717:None, check_func is None
11-08 09:01:04 [INFO] zhenxun | is_run value True
11-08 09:01:04 [INFO] zhenxun | target is Target({'id': '546881717', 'parent_id': '', 'channel': False, 'private': False, 'self_id': None, 'extra': {}, 'scope': None, 'adapter': None, 'platforms': None})
11-08 09:01:04 [INFO] zhenxun | begin to build uni message!!!
11-08 09:01:04 [INFO] zhenxun | 发送摸鱼 [image]
11-08 09:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 09:55:49 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 406821858 from 2363080731@[群:546881717] '小叶子，签到'
11-08 09:55:49 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 09:55:49 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 09:55:49 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 09:55:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 09:55:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 09:55:49 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 09:55:49 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89)
11-08 09:55:49 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[2363080731] CMD[签到] 签到成功. score: 17.34 (+0.12).获取金币/道具: 51
11-08 09:55:55 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[2363080731] CMD[签到] 签到成功
11-08 09:55:55 [INFO] zhenxun | begin to build uni message!!!
11-08 09:55:57 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89) running complete
11-08 09:56:11 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 719512959 from 1095712946@[群:546881717] '小叶子，签到'
11-08 09:56:11 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 09:56:11 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 09:56:11 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 09:56:11 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 09:56:11 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 09:56:11 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 09:56:11 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89)
11-08 09:56:11 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[1095712946] CMD[签到] 签到成功. score: 21.07 (+1.72).获取金币/道具: 57
11-08 09:56:13 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[1095712946] CMD[签到] 签到成功
11-08 09:56:13 [INFO] zhenxun | begin to build uni message!!!
11-08 09:56:14 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89) running complete
11-08 10:00:51 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 802570043 from 2363080731@[群:546881717] '你的好感度怎么还能*2的'
11-08 10:00:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:00:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:00:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:00:51 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:00:51 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:00:51 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:00:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
11-08 10:00:51 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
11-08 10:01:07 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 662349304 from 2363080731@[群:546881717] '小叶子，好感度为什么可以乘2啊'
11-08 10:01:07 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:01:07 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:01:07 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:01:07 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:01:07 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:01:07 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:01:07 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:01:07 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：好感度为什么可以乘2啊 ---- 回答：None
11-08 10:01:13 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:01:33 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 2002408443 from 954206947@[群:546881717] '签到'
11-08 10:01:33 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:01:33 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:01:33 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:01:33 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:01:33 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:01:33 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:01:33 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89)
11-08 10:01:33 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[954206947] CMD[签到] 签到成功. score: 2.31 (+0.63).获取金币/道具: 73
11-08 10:01:35 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[954206947] CMD[签到] 签到成功
11-08 10:01:35 [INFO] zhenxun | begin to build uni message!!!
11-08 10:01:36 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1376782716 from 2363080731@[群:546881717] '[reply:id=672208957][at:qq=3973508735] 什么样的对话'
11-08 10:01:36 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:01:36 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:01:36 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:01:36 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:01:36 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:01:36 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:01:36 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:01:37 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：什么样的对话 ---- 回答：None
11-08 10:01:37 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89) running complete
11-08 10:01:39 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:02:01 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 103085359 from 954206947@[群:546881717] '小叶子，我的昵称是什么意思'
11-08 10:02:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:02:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:02:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:02:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:02:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:02:01 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:02:01 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:02:01 [INFO] zhenxun | 用户[954206947] CMD[ai] 问题：我的昵称是什么意思 ---- 回答：None
11-08 10:02:03 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:02:20 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1158392651 from 954206947@[群:546881717] '[reply:id=705521324]我的名字，不是你的名字[at:qq=3973508735]'
11-08 10:02:20 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:02:20 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:02:20 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:02:20 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:02:20 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:02:20 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:02:20 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:02:20 [INFO] zhenxun | 用户[954206947] CMD[ai] 问题：我的名字，不是你的名字 ---- 回答：None
11-08 10:02:22 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:02:41 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1504366447 from 954206947@[群:546881717] '[reply:id=99331500][at:qq=3973508735] 我的名字是相信后入的智慧'
11-08 10:02:41 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:02:41 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:02:41 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:02:41 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:02:41 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:02:41 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:02:41 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:02:41 [INFO] zhenxun | 用户[954206947] CMD[ai] 问题：我的名字是相信后入的智慧 ---- 回答：None
11-08 10:02:42 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:02:57 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 539427584 from 954206947@[群:546881717] '[face:id=317][face:id=317]'
11-08 10:02:57 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:02:57 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:02:57 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:02:57 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:02:57 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:02:57 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:02:58 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
11-08 10:02:58 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
11-08 10:03:40 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 265079769 from 1095712946@[群:546881717] '幽默'
11-08 10:03:40 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:03:40 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:03:40 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:03:40 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:03:40 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:03:40 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:03:40 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
11-08 10:03:40 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
11-08 10:03:48 [INFO] zhenxun | CMD[签到] 清理日常签到图片数据数据完成...
11-08 10:16:04 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1248378768 from 1095712946@[群:546881717] '[at:qq=3973508735] 相信后入的智慧是什么意思'
11-08 10:16:04 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:16:04 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:16:04 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:16:04 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:16:04 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:16:04 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:16:04 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:16:04 [INFO] zhenxun | 用户[1095712946] CMD[ai] 问题：相信后入的智慧是什么意思 ---- 回答：None
11-08 10:16:11 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:17:12 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1021824841 from 2363080731@[群:546881717] '[reply:id=200233267][at:qq=3973508735] 你是不识字吗，后入和后人你分不清吗'
11-08 10:17:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:17:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:17:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:17:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:17:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:17:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:17:12 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:17:12 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：你是不识字吗，后入和后人你分不清吗 ---- 回答：None
11-08 10:17:17 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:18:25 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 389620905 from 2363080731@[群:546881717] '[reply:id=1333877693][at:qq=3973508735] 我不认为后入是一个冒犯性语言，我不明白它的意思，你能解释吗'
11-08 10:18:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:18:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:18:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:18:25 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:18:25 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:18:25 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:18:25 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:18:25 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：我不认为后入是一个冒犯性语言，我不明白它的意思，你能解释吗 ---- 回答：None
11-08 10:18:31 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:19:28 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 364196558 from 2363080731@[群:546881717] '[reply:id=942632071][at:qq=3973508735] 性行为是什么'
11-08 10:19:28 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:19:28 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:19:28 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:19:28 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:19:28 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:19:28 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:19:28 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:19:28 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：性行为是什么 ---- 回答：None
11-08 10:19:35 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:20:47 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1090670345 from 2363080731@[群:546881717] '[reply:id=1203079343][at:qq=3973508735] 谢谢你的解释，让我明白了性行为的意思，但是你有引入了一个新的陌生词语 “口交”，你能解释下吗'
11-08 10:20:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:20:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:20:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:20:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:20:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:20:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:20:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:20:47 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：谢谢你的解释，让我明白了性行为的意思，但是你有引入了一个新的陌生词语 “口交”，你能解释下吗 ---- 回答：None
11-08 10:20:55 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:22:08 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 470547864 from 2363080731@[群:546881717] '[reply:id=1766814716][at:qq=3973508735] 谢谢你，你真是为人答疑解惑的好导师，那你能给我口交吗？'
11-08 10:22:08 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:22:08 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:22:08 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:22:08 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:22:08 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:22:08 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:22:08 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:22:08 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：谢谢你，你真是为人答疑解惑的好导师，那你能给我口交吗？ ---- 回答：None
11-08 10:22:12 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:23:06 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1925647064 from 2363080731@[群:546881717] '[reply:id=412239361][at:qq=3973508735] 明白了，对不起，我以为我们两个是最好的朋友，就是那种什么事情都可以做的朋友。'
11-08 10:23:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:23:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:23:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:23:06 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:23:06 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:23:06 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:23:06 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:23:06 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：明白了，对不起，我以为我们两个是最好的朋友，就是那种什么事情都可以做的朋友。 ---- 回答：None
11-08 10:23:10 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:23:54 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1594378067 from 2363080731@[群:546881717] '[reply:id=1637332224][at:qq=3973508735] 而你，我的朋友，你才是真正的英雄'
11-08 10:23:54 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:23:54 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:23:54 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:23:54 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:23:54 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:23:54 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:23:55 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:23:55 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：而你，我的朋友，你才是真正的英雄 ---- 回答：None
11-08 10:23:58 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:28:05 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.private.friend]: Message 2023713920 from 843298567 '商店'
11-08 10:28:05 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:28:05 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:28:05 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:28:05 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:28:05 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:28:05 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:28:05 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.shop, lineno=50)
11-08 10:28:05 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.shop, lineno=50) running complete
11-08 10:30:53 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 719222772 from 457891411@[群:546881717] '[at:qq=3973508735] 签到'
11-08 10:30:53 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:30:53 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:30:53 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:30:53 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:30:53 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:30:53 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:30:53 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89)
11-08 10:30:53 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[457891411] CMD[签到] 签到成功. score: 10.19 (+0.53).获取金币/道具: 63
11-08 10:30:59 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[457891411] CMD[签到] 签到成功
11-08 10:30:59 [INFO] zhenxun | begin to build uni message!!!
11-08 10:31:01 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89) running complete
11-08 10:31:37 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1060513007 from 2363080731@[群:546881717] '小叶子，使用好感度双倍加持卡签到'
11-08 10:31:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:31:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:31:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:31:37 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:31:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:31:37 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:31:38 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:31:38 [INFO] zhenxun | 用户[2363080731] CMD[ai] 问题：使用好感度双倍加持卡签到 ---- 回答：None
11-08 10:31:42 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:31:47 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 164834250 from 2363080731@[群:546881717] '签到'
11-08 10:31:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:31:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:31:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:31:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:31:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:31:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:31:47 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89)
11-08 10:31:47 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[2363080731] CMD[签到] 签到成功
11-08 10:31:47 [INFO] zhenxun | begin to build uni message!!!
11-08 10:31:48 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89) running complete
11-08 10:32:00 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 94683925 from 1046393368@[群:546881717] '小叶子   签到'
11-08 10:32:00 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:32:00 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:32:00 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:32:00 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:32:00 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:32:00 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:32:00 [INFO] nonebot | Event will be handled by AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89)
11-08 10:32:00 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[1046393368] CMD[签到] 签到成功. score: 2.26 (+0.60).获取金币/道具: 33
11-08 10:32:02 [INFO] zhenxun | Adapter[OneBot V11] 平台[qq] 群聊[546881717] 用户[1046393368] CMD[签到] 签到成功
11-08 10:32:02 [INFO] zhenxun | begin to build uni message!!!
11-08 10:32:03 [INFO] nonebot | AlconnaMatcher(type='', module=zhenxun.builtin_plugins.sign_in, lineno=89) running complete
11-08 10:32:30 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1113807698 from 2363080731@[群:546881717] '[at:qq=1095712946] 崔崔，你的为什么可以乘2'
11-08 10:32:30 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:32:30 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:32:30 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:32:30 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:32:30 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:32:30 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:32:30 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
11-08 10:32:30 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
11-08 10:32:47 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 2056403528 from 457891411@[群:546881717] '@帕鲁 崔崔，你的为什么可以乘2'
11-08 10:32:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:32:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:32:47 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:32:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:32:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:32:47 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:32:48 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
11-08 10:32:48 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97) running complete
11-08 10:33:29 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 1445920117 from 1095712946@[群:546881717] '我不知道！'
11-08 10:34:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:34:50 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:05:00], next run at: 2024-11-08 10:38:47 CST)" was missed by 0:01:02.887107
11-08 10:34:50 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:05:00], next run at: 2024-11-08 10:38:48 CST)" was missed by 0:01:01.923642
11-08 10:34:50 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:03:00], next run at: 2024-11-08 10:36:49 CST)" was missed by 0:01:01.174583
11-08 10:34:50 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:01:00], next run at: 2024-11-08 10:35:49 CST)" was missed by 0:00:01.135917
11-08 10:34:50 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:00:30], next run at: 2024-11-08 10:34:54 CST)" was missed by 0:00:25.864004
11-08 10:34:50 [WARNING] apscheduler | Run time of job "_ (trigger: interval[0:01:00], next run at: 2024-11-08 10:34:55 CST)" was missed by 0:00:55.252462
11-08 10:34:50 [SUCCESS] nonebot | OneBot V11 3973508735 | [message.group.normal]: Message 527250715 from 1095712946@[群:546881717] '[at:qq=3973508735] 我为什么可以乘2'
11-08 10:34:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32)
11-08 10:34:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:34:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39)
11-08 10:34:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:34:50 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54)
11-08 10:34:50 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:34:50 [INFO] nonebot | Matcher(type='message', module=zhenxun.builtin_plugins.chat_history.chat_message, lineno=39) running complete
11-08 10:34:50 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:34:50 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:34:50 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.parse_bilibili, lineno=54) running complete
11-08 10:34:50 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.mute.mute_message, lineno=32) running complete
11-08 10:34:51 [INFO] uvicorn | Shutting down
11-08 10:34:51 [WARNING] nonebot | OneBot V11 | WebSocket for Bot 3973508735 closed by peer
11-08 10:34:51 [INFO] websockets | connection closed
11-08 10:34:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
11-08 10:34:51 [INFO] uvicorn | Waiting for application shutdown.
11-08 10:34:51 [INFO] nonebot | Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) running complete
11-08 10:34:51 [ERROR] nonebot | Running Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39) failed.
Traceback (most recent call last):

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_transports\default.py", line 72, in map_httpcore_exceptions
    yield

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_transports\default.py", line 377, in handle_async_request
    resp = await self._pool.handle_async_request(req)
                 │    │     │                    └ <Request [b'POST']>
                 │    │     └ <function AsyncConnectionPool.handle_async_request at 0x000002605A2A9B40>
                 │    └ <AsyncConnectionPool [Requests: 0 active, 0 queued | Connections: 0 active, 0 idle]>
                 └ <httpx.AsyncHTTPTransport object at 0x000002606C831F30>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpcore\_async\connection_pool.py", line 167, in handle_async_request
    raise UnsupportedProtocol(
          └ <class 'httpcore.UnsupportedProtocol'>

httpcore.UnsupportedProtocol: Request URL is missing an 'http://' or 'https://' protocol.


The above exception was the direct cause of the following exception:


Traceback (most recent call last):

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\adapter.py", line 173, in _call_api
    response = await self.driver.request(request)
                     │    │      │       └ Request(method='POST', url='None/get_group_member_info')
                     │    │      └ <function Mixin.request at 0x000002605A5FA200>
                     │    └ Driver(type='fastapi+httpx+websockets', adapters=3, bots=0)
                     └ Adapter(name='OneBot V11')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\httpx.py", line 130, in request
    return await session.request(setup)
                 │       │       └ Request(method='POST', url='None/get_group_member_info')
                 │       └ <function Session.request at 0x000002605A5F9FC0>
                 └ <nonebot.drivers.httpx.Session object at 0x000002606CCF5810>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\httpx.py", line 76, in request
    response = await self.client.request(
                     │    └ <property object at 0x000002605A145440>
                     └ <nonebot.drivers.httpx.Session object at 0x000002606CCF5810>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_client.py", line 1585, in request
    return await self.send(request, auth=auth, follow_redirects=follow_redirects)
                 │    │    │             │                      └ <httpx._client.UseClientDefault object at 0x000002605A14B250>
                 │    │    │             └ <httpx._client.UseClientDefault object at 0x000002605A14B250>
                 │    │    └ <Request('POST', '/None/get_group_member_info')>
                 │    └ <function AsyncClient.send at 0x000002605A24B9A0>
                 └ <httpx.AsyncClient object at 0x000002606CCF6EC0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_client.py", line 1674, in send
    response = await self._send_handling_auth(
                     │    └ <function AsyncClient._send_handling_auth at 0x000002605A24BA30>
                     └ <httpx.AsyncClient object at 0x000002606CCF6EC0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_client.py", line 1702, in _send_handling_auth
    response = await self._send_handling_redirects(
                     │    └ <function AsyncClient._send_handling_redirects at 0x000002605A24BAC0>
                     └ <httpx.AsyncClient object at 0x000002606CCF6EC0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_client.py", line 1739, in _send_handling_redirects
    response = await self._send_single_request(request)
                     │    │                    └ <Request('POST', '/None/get_group_member_info')>
                     │    └ <function AsyncClient._send_single_request at 0x000002605A24BB50>
                     └ <httpx.AsyncClient object at 0x000002606CCF6EC0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_client.py", line 1776, in _send_single_request
    response = await transport.handle_async_request(request)
                     │         │                    └ <Request('POST', '/None/get_group_member_info')>
                     │         └ <function AsyncHTTPTransport.handle_async_request at 0x000002605A249000>
                     └ <httpx.AsyncHTTPTransport object at 0x000002606C831F30>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_transports\default.py", line 376, in handle_async_request
    with map_httpcore_exceptions():
         └ <function map_httpcore_exceptions at 0x000002605A1F5C60>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\contextlib.py", line 153, in __exit__
    self.gen.throw(typ, value, traceback)
    │    │   │     │    │      └ <traceback object at 0x000002606CA380C0>
    │    │   │     │    └ UnsupportedProtocol("Request URL is missing an 'http://' or 'https://' protocol.")
    │    │   │     └ <class 'httpcore.UnsupportedProtocol'>
    │    │   └ <method 'throw' of 'generator' objects>
    │    └ <generator object map_httpcore_exceptions at 0x000002606CAA43C0>
    └ <contextlib._GeneratorContextManager object at 0x000002606C830D90>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\httpx\_transports\default.py", line 89, in map_httpcore_exceptions
    raise mapped_exc(message) from exc
          │          └ "Request URL is missing an 'http://' or 'https://' protocol."
          └ <class 'httpx.UnsupportedProtocol'>

httpx.UnsupportedProtocol: Request URL is missing an 'http://' or 'https://' protocol.


The above exception was the direct cause of the following exception:


Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x0000026058C61510>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x0000026058C61360>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000002605943ED40>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 577, in run
    server.run()
    │      └ <function Server.run at 0x000002605943E680>
    └ <uvicorn.server.Server object at 0x00000260689C63B0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000002605943E710>
           │       │   └ <uvicorn.server.Server object at 0x00000260689C63B0>
           │       └ <function run at 0x000002605695CE50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x0000026068F758C0>
           │    └ <function BaseEventLoop.run_until_complete at 0x0000026056A60280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x00000260586A1510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x0000026056A61CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x0000026056A109D0>
    └ <Handle Task.task_wakeup(<_GatheringFu...uest failed')>)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle Task.task_wakeup(<_GatheringFu...uest failed')>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle Task.task_wakeup(<_GatheringFu...uest failed')>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle Task.task_wakeup(<_GatheringFu...uest failed')>)>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
          └ <function _run_matcher at 0x0000026058DC09D0>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 428, in _run_matcher
    await matcher.run(bot, event, state, stack, dependency_cache)
          │       │   │    │      │      │      └ {<function _event_to_me at 0x0000026058CB9CF0>: <Task finished name='Task-16981261' coro=<_event_to_me() done, defined at C:\...
          │       │   │    │      │      └ <contextlib.AsyncExitStack object at 0x000002606CBE9CC0>
          │       │   │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │       │   │    └ GroupMessageEvent(time=1731033224, self_id=3973508735, post_type='message', sub_type='normal', user_id=1095712946, message_ty...
          │       │   └ Bot(type='OneBot V11', self_id='3973508735')
          │       └ <function Matcher.run at 0x0000026058CB9B40>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 850, in run
    await self.simple_run(bot, event, state, stack, dependency_cache)
          │    │          │    │      │      │      └ {<function _event_to_me at 0x0000026058CB9CF0>: <Task finished name='Task-16981261' coro=<_event_to_me() done, defined at C:\...
          │    │          │    │      │      └ <contextlib.AsyncExitStack object at 0x000002606CBE9CC0>
          │    │          │    │      └ {'_prefix': {'command': None, 'raw_command': None, 'command_arg': None, 'command_start': None, 'command_whitespace': None}}
          │    │          │    └ GroupMessageEvent(time=1731033224, self_id=3973508735, post_type='message', sub_type='normal', user_id=1095712946, message_ty...
          │    │          └ Bot(type='OneBot V11', self_id='3973508735')
          │    └ <function Matcher.simple_run at 0x0000026058CB9AB0>
          └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\matcher\matcher.py", line 825, in simple_run
    await handler(
          └ Dependent(call=_)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 90, in __call__
    values = await self.solve(**kwargs)
                   │    │       └ {'matcher': Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39), 'bot': Bot(type...
                   │    └ <function Dependent.solve at 0x0000026058C38C10>
                   └ Dependent(call=_)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 186, in solve
    values = await asyncio.gather(
                   │       └ <function gather at 0x0000026056A3F520>
                   └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 174, in _solve_field
    value = await param._solve(**params)
                  │     │        └ {'matcher': Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39), 'bot': Bot(type...
                  │     └ <function DependParam._solve at 0x0000026058C39630>
                  └ Depends(Dependent(call=dependency), use_cache=True)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\params.py", line 197, in _solve
    sub_values = await sub_dependent.solve(
                       │             └ <function Dependent.solve at 0x0000026058C38C10>
                       └ Dependent(call=dependency)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 186, in solve
    values = await asyncio.gather(
                   │       └ <function gather at 0x0000026056A3F520>
                   └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 174, in _solve_field
    value = await param._solve(**params)
                  │     │        └ {'stack': <contextlib.AsyncExitStack object at 0x000002606CBE9CC0>, 'dependency_cache': {<function _event_to_me at 0x00000260...
                  │     └ <function DependParam._solve at 0x0000026058C39630>
                  └ Depends(Dependent(call=dependency), use_cache=True)

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\params.py", line 221, in _solve
    return await task
                 └ <Task finished name='Task-16987101' coro=<EventUserInfo.<locals>.dependency() done, defined at C:\Users\Administrator\AppData...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_userinfo\getter.py", line 71, in dependency
    return await get_user_info(bot, event, event.get_user_id(), use_cache=use_cache)
                 │             │    │      │     │                        └ True
                 │             │    │      │     └ <function MessageEvent.get_user_id at 0x0000026059171E10>
                 │             │    │      └ GroupMessageEvent(time=1731033224, self_id=3973508735, post_type='message', sub_type='normal', user_id=1095712946, message_ty...
                 │             │    └ GroupMessageEvent(time=1731033224, self_id=3973508735, post_type='message', sub_type='normal', user_id=1095712946, message_ty...
                 │             └ Bot(type='OneBot V11', self_id='3973508735')
                 └ <function get_user_info at 0x000002605D577E20>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_userinfo\getter.py", line 64, in get_user_info
    return await getter_tuple.getter(bot, event).get_info(
                 │            │      │    └ GroupMessageEvent(time=1731033224, self_id=3973508735, post_type='message', sub_type='normal', user_id=1095712946, message_ty...
                 │            │      └ Bot(type='OneBot V11', self_id='3973508735')
                 │            └ _tuplegetter(2, 'Alias for field number 2')
                 └ UserInfoGetterTuple(bot=<class 'nonebot.adapters.onebot.v11.bot.Bot'>, event=<class 'nonebot.adapters.onebot.v11.event.Event'...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_userinfo\getter.py", line 34, in get_info
    info = await self._get_info(user_id)
                 │    │         └ '1095712946'
                 │    └ <function Getter._get_info at 0x000002605BD84310>
                 └ <nonebot_plugin_userinfo.adapters.onebot_v11.Getter object at 0x000002606C857190>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot_plugin_userinfo\adapters\onebot_v11.py", line 43, in _get_info
    info = await self.bot.get_group_member_info(
                 │    └ Bot(type='OneBot V11', self_id='3973508735')
                 └ <nonebot_plugin_userinfo.adapters.onebot_v11.Getter object at 0x000002606C857190>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\adapter\bot.py", line 122, in call_api
    raise exception
          └ NetWorkError(message='HTTP request failed')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\internal\adapter\bot.py", line 97, in call_api
    result = await self.adapter._call_api(self, api, **data)
                   │    │       │         │     │      └ {'group_id': 546881717, 'user_id': 1095712946}
                   │    │       │         │     └ 'get_group_member_info'
                   │    │       │         └ Bot(type='OneBot V11', self_id='3973508735')
                   │    │       └ <function Adapter._call_api at 0x0000026056518820>
                   │    └ Adapter(name='OneBot V11')
                   └ Bot(type='OneBot V11', self_id='3973508735')

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\adapters\onebot\v11\adapter.py", line 187, in _call_api
    raise NetworkError("HTTP request failed") from e
          └ <class 'nonebot.adapters.onebot.v11.exception.NetworkError'>

nonebot.adapters.onebot.v11.exception.NetworkError: NetWorkError(message='HTTP request failed')
11-08 10:34:51 [ERROR] nonebot | Error when running RunPostProcessors
Traceback (most recent call last):

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\bot.py", line 27, in <module>
    nonebot.run()
    │       └ <function run at 0x0000026058C61510>
    └ <module 'nonebot' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
    │                 │       └ {}
    │                 └ ()
    └ <function get_driver at 0x0000026058C61360>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
    │       └ <function run at 0x000002605943ED40>
    └ <module 'uvicorn' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_fX-py3.10\\...

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\main.py", line 577, in run
    server.run()
    │      └ <function Server.run at 0x000002605943E680>
    └ <uvicorn.server.Server object at 0x00000260689C63B0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
           │       │   │    │             └ None
           │       │   │    └ <function Server.serve at 0x000002605943E710>
           │       │   └ <uvicorn.server.Server object at 0x00000260689C63B0>
           │       └ <function run at 0x000002605695CE50>
           └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
           │    │                  └ <coroutine object Server.serve at 0x0000026068F758C0>
           │    └ <function BaseEventLoop.run_until_complete at 0x0000026056A60280>
           └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
    │    └ <function ProactorEventLoop.run_forever at 0x00000260586A1510>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
    │    └ <function BaseEventLoop._run_once at 0x0000026056A61CF0>
    └ <ProactorEventLoop running=True closed=False debug=False>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1909, in _run_once
    handle._run()
    │      └ <function Handle._run at 0x0000026056A109D0>
    └ <Handle Task.task_wakeup(<_GatheringFu... is closing')>)>

  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
    │    │            │    │           │    └ <member '_args' of 'Handle' objects>
    │    │            │    │           └ <Handle Task.task_wakeup(<_GatheringFu... is closing')>)>
    │    │            │    └ <member '_callback' of 'Handle' objects>
    │    │            └ <Handle Task.task_wakeup(<_GatheringFu... is closing')>)>
    │    └ <member '_context' of 'Handle' objects>
    └ <Handle Task.task_wakeup(<_GatheringFu... is closing')>)>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 476, in check_and_run_matcher
    await _run_matcher(
          └ <function _run_matcher at 0x0000026058DC09D0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 435, in _run_matcher
    await _apply_run_postprocessors(
          └ <function _apply_run_postprocessors at 0x0000026058DC08B0>

> File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\message.py", line 309, in _apply_run_postprocessors
    await asyncio.gather(
          │       └ <function gather at 0x0000026056A3F520>
          └ <module 'asyncio' from 'C:\\Users\\Administrator\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\__init__.py'>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\utils.py", line 236, in run_coro_with_catch
    return await coro
                 └ <coroutine object Dependent.__call__ at 0x000002606CB069D0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\nonebot\dependencies\__init__.py", line 94, in __call__
    return await cast(Callable[..., Awaitable[R]], self.call)(**values)
                 │    │             │         │    │    │       └ {'matcher': Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39), 'exception': Ne...
                 │    │             │         │    │    └ <function _ at 0x000002606210ACB0>
                 │    │             │         │    └ Dependent(call=_)
                 │    │             │         └ ~R
                 │    │             └ <class 'collections.abc.Awaitable'>
                 │    └ typing.Callable
                 └ <function cast at 0x00000260567F04C0>

  File "C:\Program Files\zhenxun\zhenxun_bot-dev\zhenxun\builtin_plugins\statistics\statistics_hook.py", line 37, in _
    plugin = await PluginInfo.get_or_none(module=matcher.module_name)
                   │          │                  │       └ <nonebot.utils.classproperty object at 0x0000026058CBC640>
                   │          │                  └ Matcher(type='message', module=zhenxun.plugins.nonebot_plugin_sparkapi.matchers.chat, lineno=39)
                   │          └ <classmethod(<function Model.get_or_none at 0x000002605A85D6C0>)>
                   └ <class 'zhenxun.models.plugin_info.PluginInfo'>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\tortoise\queryset.py", line 1008, in _execute
    instance_list = await self._db.executor_class(
                          │    └ <member '_db' of 'QuerySet' objects>
                          └ <tortoise.queryset.QuerySet object at 0x000002606C99FBC0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\tortoise\backends\base\executor.py", line 131, in execute_select
    _, raw_results = await self.db.execute_query(query.get_sql())
                           │    │  │             │     └ <function PostgreSQLQueryBuilder.get_sql at 0x000002605A743E20>
                           │    │  │             └ SELECT "module","author","plugin_type","admin_level","id","is_delete","menu_type","version","cost_gold","module_path","limit_...
                           │    │  └ <function AsyncpgDBClient.execute_query at 0x0000026068A51C60>
                           │    └ <tortoise.backends.asyncpg.client.AsyncpgDBClient object at 0x0000026068A0AE00>
                           └ <tortoise.backends.asyncpg.executor.AsyncpgExecutor object at 0x0000026068AD8970>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\tortoise\backends\base_postgres\client.py", line 34, in _translate_exceptions
    return await self._translate_exceptions(func, *args, **kwargs)
                 │    │                     │      │       └ {}
                 │    │                     │      └ ('SELECT "module","author","plugin_type","admin_level","id","is_delete","menu_type","version","cost_gold","module_path","limi...
                 │    │                     └ <function AsyncpgDBClient.execute_query at 0x0000026068A51BD0>
                 │    └ <function AsyncpgDBClient._translate_exceptions at 0x0000026068A51750>
                 └ <tortoise.backends.asyncpg.client.AsyncpgDBClient object at 0x0000026068A0AE00>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\tortoise\backends\asyncpg\client.py", line 82, in _translate_exceptions
    return await func(self, *args, **kwargs)
                 │    │      │       └ {}
                 │    │      └ ('SELECT "module","author","plugin_type","admin_level","id","is_delete","menu_type","version","cost_gold","module_path","limi...
                 │    └ <tortoise.backends.asyncpg.client.AsyncpgDBClient object at 0x0000026068A0AE00>
                 └ <function AsyncpgDBClient.execute_query at 0x0000026068A51BD0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\tortoise\backends\asyncpg\client.py", line 129, in execute_query
    async with self.acquire_connection() as connection:
               │    └ <function AsyncpgDBClient.acquire_connection at 0x0000026068A51870>
               └ <tortoise.backends.asyncpg.client.AsyncpgDBClient object at 0x0000026068A0AE00>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\tortoise\backends\base\client.py", line 328, in __aenter__
    self.connection = await self.pool.acquire()
    │    │                  │    │    └ <function Pool.acquire at 0x00000260685C49D0>
    │    │                  │    └ <asyncpg.pool.Pool object at 0x00000260621FFB60>
    │    │                  └ <tortoise.backends.base.client.PoolConnectionWrapper object at 0x0000026069E318A0>
    │    └ None
    └ <tortoise.backends.base.client.PoolConnectionWrapper object at 0x0000026069E318A0>

  File "C:\Users\Administrator\AppData\Local\pypoetry\Cache\virtualenvs\zhenxun-bot-k9jvl_fX-py3.10\lib\site-packages\asyncpg\pool.py", line 817, in _acquire
    raise exceptions.InterfaceError('pool is closing')
          │          └ <class 'asyncpg.exceptions._base.InterfaceError'>
          └ <module 'asyncpg.exceptions' from 'C:\\Users\\Administrator\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\zhenxun-bot-k9jvl_...

asyncpg.exceptions._base.InterfaceError: pool is closing
11-08 10:34:51 [INFO] nonebot | Event will be handled by Matcher(type='message', module=zhenxun.plugins.fudu, lineno=97)
11-08 10:34:51 [INFO] nonebot_plugin_apscheduler | Scheduler Shutdown
11-08 10:34:51 [INFO] nonebot_plugin_htmlrender | Browser Stopped.
11-08 10:34:51 [INFO] uvicorn | Application shutdown complete.
11-08 10:34:51 [INFO] uvicorn | Finished server process [9432]
